packages/components/src/components/form/Select/Select.tsx

```diff
@@ -217,6 +217,8 @@ const Wrapper = styled.div<WrapperProps>`
         border: none;
         z-index: ${zIndices.base};
     }
+
+    ${({ $isDisabled }) => $isDisabled && `pointer-events: auto; cursor: not-allowed;`};
```

Comment:
This maybe works but I think it should be: 

```ts
    ${({ $isDisabled }) =>
        $isDisabled &&
        css`
            pointer-events: auto;
            cursor: not-allowed;
        `};
```

Or at least we use it this way everywhere else. Also with this `css` auto-format works and it then places every CSS directive on separate line.

Reply:
Right! And it's more readable like this. Don't know why I've used that one line format here instead :)